src/applications/SurvivalPredictor.h

```diff
@@ -861,115 +860,135 @@ std::vector<typename ImageType::Pointer> SurvivalPredictor::RevisedTumorArea(con
+      roi21It.Set(301);
+    else if (atlasIt.Get() == 100 || atlasIt.Get() == 125 || atlasIt.Get() == 130 || atlasIt.Get() == 139 || atlasIt.Get() == 140 || atlasIt.Get() == 145 || atlasIt.Get() == 165 || atlasIt.Get() == 21 || atlasIt.Get() == 26 || atlasIt.Get() == 36 || atlasIt.Get() == 59 || atlasIt.Get() == 86 || atlasIt.Get() == 99)
+      roi21It.Set(302);
+    else if (atlasIt.Get() == 101 || atlasIt.Get() == 118 || atlasIt.Get() == 119 || atlasIt.Get() == 18 || atlasIt.Get() == 196 || atlasIt.Get() == 61 || atlasIt.Get() == 62 || atlasIt.Get() == 64 || atlasIt.Get() == 83 || atlasIt.Get() == 92 || atlasIt.Get() == 94 || atlasIt.Get() == 95 || atlasIt.Get() == 96)
+      roi21It.Set(303);
+    else if (atlasIt.Get() == 102 || atlasIt.Get() == 33 || atlasIt.Get() == 72)
+      roi21It.Set(304);
+    else if (atlasIt.Get() == 105 || atlasIt.Get() == 107 || atlasIt.Get() == 110 || atlasIt.Get() == 19 || atlasIt.Get() == 32 || atlasIt.Get() == 60 || atlasIt.Get() == 88)
+      roi21It.Set(305);
+    else if (atlasIt.Get() == 108)
+      roi21It.Set(306);
+    else if (atlasIt.Get() == 11 || atlasIt.Get() == 128 || atlasIt.Get() == 16 || atlasIt.Get() == 35 || atlasIt.Get() == 53)
+      roi21It.Set(307);
+    else if (atlasIt.Get() == 112 || atlasIt.Get() == 132 || atlasIt.Get() == 175 || atlasIt.Get() == 38 || atlasIt.Get() == 45 || atlasIt.Get() == 63 || atlasIt.Get() == 97)
+      roi21It.Set(308);
+    else if (atlasIt.Get() == 12 || atlasIt.Get() == 14 || atlasIt.Get() == 34 || atlasIt.Get() == 39 || atlasIt.Get() == 43)
+      roi21It.Set(309);
+    else if (atlasIt.Get() == 133)
+      roi21It.Set(310);
+    else if (atlasIt.Get() == 136 || atlasIt.Get() == 15 || atlasIt.Get() == 30 || atlasIt.Get() == 50 || atlasIt.Get() == 70 || atlasIt.Get() == 80 || atlasIt.Get() == 85 || atlasIt.Get() == 9 || atlasIt.Get() == 90)
+      roi21It.Set(311);
+    else if (atlasIt.Get() == 137 || atlasIt.Get() == 159 || atlasIt.Get() == 41 || atlasIt.Get() == 52 || atlasIt.Get() == 56 || atlasIt.Get() == 57 || atlasIt.Get() == 74)
+      roi21It.Set(312);
+    else if (atlasIt.Get() == 154 || atlasIt.Get() == 251 || atlasIt.Get() == 37 || atlasIt.Get() == 54 || atlasIt.Get() == 69 || atlasIt.Get() == 73 || atlasIt.Get() == 98)
+      roi21It.Set(313);
+    else if (atlasIt.Get() == 20)
+      roi21It.Set(314);
+    else if (atlasIt.Get() == 203 || atlasIt.Get() == 23 || atlasIt.Get() == 25)
+      roi21It.Set(315);
+    else if (atlasIt.Get() == 254)
+      roi21It.Set(316);
+    else if (atlasIt.Get() == 27)
+      roi21It.Set(317);
+    else if (atlasIt.Get() == 29)
+      roi21It.Set(318);
+    else if (atlasIt.Get() == 4)
+      roi21It.Set(319);
+    else if (atlasIt.Get() == 67 || atlasIt.Get() == 76)
+      roi21It.Set(320);
+    else if (atlasIt.Get() == 7)
+      roi21It.Set(321);
```

Comment:
What are the hard-coded values? Needs comments.

Reply:
these hard-coded values have been taken from the Matlab code. we may read these mapping from a csv in future to improve the code in the function.